src/services/chirpstack/chirpstack-gateway.service.ts

```diff
@@ -112,6 +112,11 @@ export class ChirpstackGatewayService extends GenericChirpstackConfigurationServ
         };
     }
 
+    /**
+     * Fetch gateways individually. This gives us the tags which contain the OS2 organization id.
+     * This is a very expensive operation, but it's the only way to retrieve gateway tags.
```

Comment:
If this turns out to be a problem, we will have to look into having a Gateway entity in the OS2iot database

Reply:
I agree.